src/main/java/org/jvnet/hudson/test/InboundAgentRule.java

```diff
@@ -95,7 +95,7 @@ public static final class Options implements Serializable {
         private boolean webSocket;
         @CheckForNull private String tunnel;
         private boolean start = true;
-        private final Map<String, Level> loggers = new LinkedHashMap<>();
+        private final LinkedHashMap<String, Level> loggers = new LinkedHashMap<>();
```

Comment:
An interesting case for language nerds: we need to prove to the compiler that the map is `Serializable` in https://github.com/jenkinsci/jenkins-test-harness/blob/7a4133525c7591d025da037fd2ed3c19a34c9e8c/src/main/java/org/jvnet/hudson/test/InboundAgentRule.java#L315 which works even though the method definition https://github.com/jenkinsci/jenkins-test-harness/blob/7a4133525c7591d025da037fd2ed3c19a34c9e8c/src/main/java/org/jvnet/hudson/test/InboundAgentRule.java#L471 declares a non-`Serializable` argument type.